Gopkg.lock

```diff
@@ -1108,11 +1108,14 @@
   version = "kubernetes-1.12.6"
 
 [[projects]]
-  digest = "1:b8a1dcc5f4e559b7af185ba12dd341cb8c175ea3d36227a02699b251ae5fde05"
+  digest = "1:51f330b7a61ac0e8b6e082490586ff2b18008eb514a67eccbe5ec78ffd32ee25"
```

Comment:
Interesting that this changed... I guess because the generated clients were updated?

Reply:
Right, I assume it's because the actual `pkg/client/clientset/*` packages are now listed (among others).  To be sure it was required I nuked the Gopkg.lock changes locally and re-ran `hack/update-codegen.sh`.  When the same changes were produced I opted to include in the PR.

Reply:
Thanks for checking that!